src/common/snippets/CMakeLists.txt

```diff
@@ -4,10 +4,14 @@
 
 set (TARGET_NAME "inference_engine_snippets")
 
+if (ENABLE_CPU_DEBUG_CAPS_SNIPPETS)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DCPU_DEBUG_CAPS_SNIPPETS")
+endif()
```

Comment:
do we need to introduce new option for each module? We already have similar flags for CPU / GPU

Reply:
it was just to debug in draft PR : ) 
was removed, thanks!